crates/cast/tests/cli/main.rs

```diff
@@ -2003,7 +2003,16 @@ contract LocalProjectScript is Script {
         .stdout_eq(str![[r#"
 Executing previous transactions from the block.
 Compiling project to generate artifacts
-Nothing to compile
```

Comment:
Hm, can you share in what conditions do you hit this? The cast call is be done from outside of the project hence `Nothing to compile` should be the right output. thanks!